samza-kv/src/main/scala/org/apache/samza/storage/kv/LoggedStore.scala

```diff
@@ -114,7 +128,14 @@ class LoggedStore[K, V](
     store.close
   }
 
-  override def snapshot(from: K, to: K): KeyValueSnapshot[K, V] = {
+  override def snapshot(from: Array[Byte], to: Array[Byte]): KeyValueSnapshot[Array[Byte], Array[Byte]] = {
     store.snapshot(from, to)
   }
+
+  private def validateMessageSize(key: Array[Byte], message: Array[Byte]): Unit = {
+    if (message.length > maxMessageSize) {
+      throw new SamzaException("RocksDB message size " + message.length + " for store " + storeName
```

Comment:
nit: exception message is for the logged store, not RocksDB. AFAIK RocksDB has no limit.

Reply:
Done.